code-challenges.test.js

```diff
@@ -56,3 +150,34 @@ const hand4 = [7, 2, 7, 2, 7]
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// Create a function that accepts an array as parameter
+// Create an empty object
+// Loop through the array
+    //if object key is not defined in object
+        //give key a value of 1
+    //otherwise
+        //increment value
+// Convert the values of object into an array using Object.values and sort it ascending
+// Return true if first number in array is 2 and second number is 3
+    //otherwise return false
+
+
+const fullHouse = (arr) => {
+    let obj = {}
+    for (let num of arr){
+        if (obj[num] === undefined){
+            obj[num] = 1;
+        }else{
+            obj[num]++
+        }
+    }
+    let res = Object.values(obj).sort()
+    if (res[0] === 2 && res[1] === 3){
+        return true;
+    }else{
+        return false;
+    }
+}
+
```

Comment:
Great work on these functions, Peter!  Well thought out, and your pseudocode is excellent.  Although you did a great job with the 3 one and using Objects and Object.value, I would encourage you to give it a try with some of the more fundamental concepts.  I think you did a great job, though.  You are definitely showing a competent skill level! 🎉